src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -312,6 +322,14 @@ public static string GetAzureWebsiteUniqueSlotName(this IEnvironment environment
             return name?.ToLowerInvariant();
         }
 
+        /// <summary>
+        /// Gets a value from AzureWebsiteName environment variables.
+        /// </summary>
+        public static string GetKubernetesHostname(this IEnvironment environment)
+        {
+            return environment.GetEnvironmentVariable(AzureWebsiteName);
```

Comment:
We don't add this currently in core-tools, right?

Reply:
Yes. we don't have it.  current environment variables when I deploy with `func kubernetes deploy` is 

```
root@timger-func-68bc867dd8-gvhx8:/# env
WEBSITE_SITE_NAME=singletonTimer
KUBERNETES_SERVICE_PORT_HTTPS=443
KUBERNETES_SERVICE_PORT=443
HOSTNAME=timger-func-68bc867dd8-gvhx8
AzureFunctionsJobHost__Logging__Console__IsEnabled=true
ASPNETCORE_URLS=http://+:80
PWD=/
AzureWebJobsScriptRoot=/home/site/wwwroot
FUNCTIONS_WORKER_RUNTIME=dotnet
HOME=/home
KUBERNETES_PORT_443_TCP=tcp://10.0.0.1:443
AzureFunctionsJobHost__functions__0=TimerTrigger
AzureWebJobsStorage=MY_CONNECTION_STRING
TERM=xterm
HOST_VERSION=3.0.15277
SHLVL=1
KUBERNETES_PORT_443_TCP_PROTO=tcp
DOTNET_RUNNING_IN_CONTAINER=true
DOTNET_USE_POLLING_FILE_WATCHER=true
KUBERNETES_PORT_443_TCP_ADDR=10.0.0.1
KUBERNETES_SERVICE_HOST=10.0.0.1
KUBERNETES_PORT=tcp://10.0.0.1:443
KUBERNETES_PORT_443_TCP_PORT=443
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
_=/usr/bin/env
``